dallinger/experiment.py

```diff
@@ -967,6 +967,24 @@ def monitoring_statistics(self, **kw):
 
         return stats
 
+    @staticmethod
+    def count_active_complete_failed(entities):
+        n_pending = 0
+        n_completed = 0
+        n_failed = 0
+        for entity in entities:
+            if entity.failed is False:
```

Comment:
This is a bit strange, normally you would just say `if not entity.failed`, right?

Reply:
Surprised that it let you merge without resolving this comment